jellyfin_kodi/jellyfin/ws_client.py

```diff
@@ -62,23 +61,18 @@ def run(self):
 
         while not self.stop:
 
-            time.sleep(self.retry_count * 5)
-            self.wsc.run_forever(ping_interval=10)
+            self.wsc.run_forever(ping_interval=10, reconnect=10)
```

Comment:
Looks like the main functional change in this PR is the addition of the reconnect parameter here.

Could you elaborate on the reasoning behind removing the back-off delay?

Reply:
The reconnect param makes the reconnect logic handled by the websocket library. I added logging to the backoff delay and found it wasn't getting called anymore so I removed it for clarity.